tests/integration/docker/user.bats

```diff
@@ -45,3 +45,22 @@ setup() {
 	[ "${status}" -eq 0 ]
 	[[ "${output}" == 'root'* ]]
 }
+
+@test "Run with additional groups" {
+	run $DOCKER_EXE run --rm --group-add audio --group-add nogroup busybox id
```

Comment:
If the user provides a group name [not a gid], that is non-existent, docker itself handles that and throws this error "linux spec user: Unable to find group". If its a non-existent gid, then that is expected to be added by the runtime. Thats the reason I did not add test for the former scenario(non-existent group) but for the latter(non-existent gid).